packages/p2p-v2/src/components/UserAvatar/UserAvatar.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { getShortNickname } from '../../utils/string';
+import { TGenericSizes } from '../../utils/types';
+import './UserAvatar.scss';
+
+type TUserAvatarProps = {
+    className?: string;
+    nickname: string;
+    size: number;
+    textSize: TGenericSizes;
+};
+
+const UserAvatar = memo(({ className, nickname, size, textSize }: TUserAvatarProps) => {
```

Comment:
I've added `Avatar` component as well, you can add a TODO to replace mine with yours at `components/AdvertiserName` since it has more variation to it